packages/intrepid2/src/Orientation/Intrepid2_OrientationTools.hpp

```diff
@@ -369,104 +370,105 @@ namespace Intrepid2 {
+  template<typename DeviceType>
   class OrientationTools {
   public:
+
     /** \brief  subcell ordinal, orientation, matrix m x n
-    */
-    typedef Kokkos::View<double****,ExecSpaceType> CoeffMatrixDataViewType;
+     */
+    typedef Kokkos::View<double****,DeviceType> CoeffMatrixDataViewType;
 
     // 
     /** \brief  key :: basis name, order, value :: matrix data view type 
-    */
+     */
     static std::map<std::pair<std::string,ordinal_type>,CoeffMatrixDataViewType> ortCoeffData;
     
   private:
 
-    template<typename BasisType>
+    template<typename BasisHostType>
     inline 
-    static CoeffMatrixDataViewType createCoeffMatrixInternal(const BasisType* basis);
+    static CoeffMatrixDataViewType createCoeffMatrixInternal(const BasisHostType* basis);
     
 
     /** \brief  Compute orientation matrix for HGRAD basis
-        */
-    template<typename BasisType>
+     */
+    template<typename BasisHostType>
     inline
     static void init_HGRAD(CoeffMatrixDataViewType matData,
-                               BasisType const *cellBasis);
+                           BasisHostType const *cellBasis);
 
     /** \brief  Compute orientation matrix for HCURL basis
-        */
-    template<typename BasisType>
+     */
+    template<typename BasisHostType>
     inline
     static void init_HCURL(CoeffMatrixDataViewType matData,
-        BasisType const *cellBasis);
+                           BasisHostType const *cellBasis);
 
     /** \brief  Compute orientation matrix for HDIV basis
-        */
-    template<typename BasisType>
+     */
+    template<typename BasisHostType>
     inline
     static void init_HDIV(CoeffMatrixDataViewType matData,
-        BasisType const *cellBasis);
+                          BasisHostType const *cellBasis);
     
   public:
 
     /** \brief  Create coefficient matrix.
-          \param  basis      [in]  - basis type
+        \param  basis      [in]  - basis type
     */
     template<typename BasisType>
     inline 
     static CoeffMatrixDataViewType createCoeffMatrix(const BasisType* basis);
 
     /** \brief  Clear coefficient matrix
-    */
+     */
     inline 
     static void clearCoeffMatrix();
 
     /** \brief  Compute orientations of cells in a workset
-          \param  elemOrts      [out]  - cell orientations
-          \param  elemNodes      [in]  - node coordinates
-          \param  cellTopo       [in]  - shards cell topology
+        \param  elemOrts      [out]  - cell orientations
+        \param  elemNodes      [in]  - node coordinates
+        \param  cellTopo       [in]  - shards cell topology
     */
     template<typename elemOrtValueType, class ...elemOrtProperties,
              typename elemNodeValueType, class ...elemNodeProperties>
     inline 
     static void
-    getOrientation(      Kokkos::DynRankView<elemOrtValueType,elemOrtProperties...> elemOrts,
+    getOrientation(/* */ Kokkos::DynRankView<elemOrtValueType,elemOrtProperties...> elemOrts,
```

Comment:
I remember Kara saying that `/* */` messes up with Doxygen. Have you checked whether this didn't disrupt Doxygen?

Reply:
done.